index.php

```diff
@@ -58,19 +58,18 @@
         $data['content']['customerDetails'] = $reserSys->showCustomer($_POST['postCustomerId'], $_POST['postChannelId']);
     }
 } elseif ($page == 'bookingDetails') {
-    if (isset($_POST['postUpdateCancelation'])) {
-        $cancelation = $reserSys->cancelBooking($_POST['postChannelId'], $_POST['postBookingId'], $_POST['postCancelationReason']);
-        $data['content']['bookingDetails'] = $reserSys->forceShowBookingUpdate($_POST['postChannelId'], $_POST['postBookingId']);
-        generalFormat($data, $data['content']['bookingDetails']);
-    } elseif (isset($_POST['postSearchBooking'])) {
-        try {
-            $data['content']['bookingDetails'] = $reserSys->showBooking($_POST['postChannelId'], $_POST['postBookingId']);
-            $data['content']['bookingDetailsError'] = false;
-            generalFormat($data, $data['content']['bookingDetails']);
-        } catch (Exception $e) {
-            $data['content']['bookingDetailsError'] = $e->getMessage();
+    if (isset($_POST['postSearchBooking'])) {
+        if (isset($_POST['postUpdateCancelation'])) {
+            $cancelation = $reserSys->cancelBooking($_POST['postChannelId'], $_POST['postBookingId'], $_POST['postCancelationReason']);
         }
+        if(isset($_POST['postCreatePayment'])){
+            $payment = $reserSys->createPayment($_POST['postChannelId'], $_POST['postInfoPayment']);
+            }
+        $data['content']['bookingDetails'] = $reserSys->showBooking($_POST['postChannelId'], $_POST['postBookingId']);
+        $data['content']['bookingDetails']['auxBooleanPaymentFulfilled'] =  $data['content']['bookingDetails']['payment_status'] == 3 ? 1 : 0;
```

Comment:
what is 3?
Please use user friendly const instead of magic number / words

Reply:
fixed